Tests/swift-sdk-swift-tests/InboxImpressionTrackerTests.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  InboxImpressionTrackerTests.swift
+//  swift-sdk-swift-tests
+//
```

Comment:
nit: Please remove these first two lines. They don't really help much and they will be confusing if we change the name of file.